test/test_helper.rb

```diff
@@ -0,0 +1,34 @@
+require 'coveralls'
+Coveralls.wear!('rails')
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+
+ENV["RAILS_ENV"] ||= "test"
+
+# This adds the current file directory to the load path.
+$:.unshift File.dirname(__FILE__)
+
+require 'config/application'
+require 'rails/test_help'
+require "database_cleaner"
+require "mongoid"
+require 'minitest/reporters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.